frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/state.ts

```diff
@@ -16,7 +16,7 @@ export const initialState: State = {
   showNodesListOnLVS: false,
   diskType: 'All',
   diskMode: diskModeDropdownItems.BLOCK,
-  deviceType: [],
+  deviceType: ['Disk', 'Part'],
```

Comment:
Use the `deviceTypeDropdownItems`
```suggestion
  deviceType: [deviceTypeDropdownItems.DISK, deviceTypeDropdownItems.PART],
```
`

Reply:
@afreen23 Sure. Will do it.

Reply:
This is to ensure that the changes remain consistent all over in case of modifications